### ISSUE TITLE
Cannot get an image to render in the header.

### ISSUE THREAD

**Issue by chapeljuice** [react-community/react-navigation#1342]
### Current Behavior
I cannot get an image to render in my navigation header.

I've tried setting the values of 'title', 'headerTitle', and 'headerLeft' to my Image element, with no success.
<img width="678" alt="screen shot 2017-05-02 at 11 42 32 am" src="https://cloud.githubusercontent.com/assets/862642/25633800/35633bd2-2f2d-11e7-8e40-5b90d9b7a8c7.png">

It always just shows the string in 'title'.
<img width="556" alt="screen shot 2017-05-02 at 11 42 53 am" src="https://cloud.githubusercontent.com/assets/862642/25633813/3ca49594-2f2d-11e7-907b-62a20bab16ca.png">

### Expected Behavior
Shouldn't this display my Image?

### Your Environment
Android 7.1.1 emulator on a 2016 15" MBP w/ touch bar.

| software         | version
| ---------------- | -------
| react-navigation |  ^1.0.0-beta.7
| react-native     |  0.42.0
| node             | v7.9.0
| npm       | v4.2.0
| yarn       | 0.23.2
| ignite       | 2.0.0-beta.8

**Comment by koenpunt** [comment]
You must be using beta.9 for the new "flat options". See release notes here: https://github.com/react-community/react-navigation/releases/tag/v1.0.0-beta.9

**Comment by chapeljuice** [comment]
I don't see anything in those release notes pertaining to my issue. Is there something I missed?

**Comment by koenpunt** [comment]
You description states; `1.0.0-beta.7`, but you have to be on `1.0.0-beta.9`, because before that the `headerTitle` option did not exist as such.

For `1.0.0-beta.7` it was something like;

```js
navigationOptions: {
  header: {
    title: <Image ... />,
  }
}
```

**Comment by chapeljuice** [comment]
Okay ... apologies if this isn't the place for this, but I tried to update to `1.0.0-beta.9` and in the process seems as though I've really mucked up my project. :/

Any clue how to resolve this?
<img width="544" alt="screen shot 2017-05-02 at 3 35 26 pm" src="https://cloud.githubusercontent.com/assets/862642/25642234/66f2afe6-2f4d-11e7-87f3-e81603fe1039.png">

**Comment by annelorraineuy** [comment]
@chapeljuice maybe use `react-native log-android` to see more info about the error log.

**Comment by jeffreymendez1993** [comment]
close packager and restart it and your issue should be fixed.

**Comment by ofairfoul** [comment]
@chapeljuice terminate and then start the pacakger with the following arguments
`react-native start --reset-cache`

**Comment by chapeljuice** [comment]
Wow, so thank you @jeffreymendez1993 and @ofairfoul , that was indeed the fix for that issue.

A bit of an aside - but maybe upgrading to newer versions needs to be easier or more straight-forward? After I finally got that error taken care of I upgraded to `1.0.0-beta.9` and had a whole other bunch of problems until I realized that the older, "non-flat" styling syntax was breaking everything.

Anyhoo, thanks everyone for the help. I really appreciate it!
